project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
   object V {
     // Java
     val config           = "1.4.1"
-    val slf4j            = "1.7.32"
+    val slf4j            = "2.0.6"
```

Comment:
Have you testing that logging still works after bumping the major? I think that I've already had an issue with that

Reply:
Yes it is tested.  Logging still works.